### PR TITLE
Extended hello-sample

### PR DESCRIPTION
**Pull request by lechnerc77**
This pull request contains the extension of the hello-sample GitHub page section with the information necessary to build, run and deploy the new samples introduced in pull request <https://github.com/microsoft/durabletask-netherite/pull/72>

### CODE REVIEW

File: docs/hello-sample.md [hunk 1]
```diff
@@ -1,150 +1,300 @@
-# Running the 'Hello' Sample
+# Running the 'Hello' Samples
 
-Here are simple instructions for running the 'Hello' sample project. We included scripts that make it easy to build, run, and deploy this application. Also, this sample is a great resource for understanding how Durable Functions and Netherite work, and an excellent starting point for creating your own projects.
+In order to speed up your first experience with Netherite we provide some samples that serve this goal. They are also a great resource for understanding how Durable Functions and Netherite work. They are also excellent starting point for creating your own projects.
```
Comment (sebastianburckhardt) [inline docs/hello-sample.md:3]: Avoid starting sentence with "They are also" twice in a row.

File: docs/hello-sample.md [hunk 2]
```diff
@@ -1,150 +1,300 @@
...
+
+2. [Azure Functions Core Tools 3.x](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=windows%2Ccsharp%2Cbash) or later. You can run `func --version` to check what is installed.
+
+3. [The Azure CLI 2.18.0](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) or later. You can run `az --version` to check what is installed.
+
+4. [PowerShell 7.1](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-7.1) or later. You can check what is installed with `pwsh --version`. PowerShell is easy to install or update via `dotnet tool install --global PowerShell` or `dotnet tool install --global PowerShell`, respectively.
+
+5. [Azurite 3.x](https://github.com/Azure/Azurite) as storage emulator for the local execution. You can run `azurite --version` to check what is installed.
+
+6. [An Azure subscription](https://azure.microsoft.com/free/search/), to allocate and deploy the required resources.
```
Comment (sebastianburckhardt) [inline docs/hello-sample.md:29]: Maybe you could mention here that it is possible to run the samples local-only without an Azure account?

File: docs/hello-sample.md [hunk 3]
```diff
@@ -1,150 +1,300 @@
...
+
+To execute the REST calls we recommend to use the [RESTClient for VSCode](https://marketplace.visualstudio.com/items?itemName=humao.rest-client).
+
```
Comment (sebastianburckhardt) [inline docs/hello-sample.md:32]: Interesting, I did not know about this, will try it out!

File: docs/hello-sample.md [hunk 4]
```diff
@@ -1,150 +1,300 @@
...
 
 ## Get it and build it
 
-The sample is in the directory [/samples/hello](https://github.com/microsoft/durabletask-netherite/tree/main/samples/Hello). You can download just this folder if you wish, or clone the entire repository. Note that PowerShell sometimes refuses to execute downloaded files; so using git clone may work better.
+The samples are in the directory [/samples/](https://github.com/microsoft/durabletask-netherite/tree/main/samples). You can download just this folder if you wish, or clone the entire repository. Note that PowerShell sometimes refuses to execute downloaded files so using git clone may work better.
+
+The language-specific folder in the `samples` folder contain "minimal" Azure Durable Functions projects (much like the [DF Quick Start](https://docs.microsoft.com/azure/azure-functions/durable/durable-functions-create-first-csharp?pivots=code-editor-visualstudio)).
+
+⚠ **CAUTION:** Be aware that all projects have no authentication in place. This makes the calling of the Functions easy, but this is not feasible setup for a production environment.
```
Comment (sebastianburckhardt) [inline docs/hello-sample.md:59]: Great!

File: docs/hello-sample.md [hunk 5]
```diff
@@ -1,150 +1,300 @@
...
-- a single file of code, `HelloCities.cs`, which defines three functions: a trigger, an orchestration, and an activity.
-- a configuration file `host.json` which contains settings.
-- a configuration file `local.settings.json` which contains settings for local testing.
+- the single file of code, `HelloCities.cs`, which defines three functions: a trigger, an orchestration, and an activity.
+- the configuration file `host.json` which contains settings.
+- the configuration file `local.settings.json` which contains settings for local execution.
 
-Additionally, the folder contains six scripts (files ending in *.ps1) that make it easy to perform typical operations (init, run, deploy, clear, delete) using automated CLI commands. These scripts are purely optional; one can perform all the steps in these scripts manually, using the Azure CLI or the Azure Portal.
+Additionally, the folder contains PowerShell scripts (files ending in `*.ps1`) that make it easy to run amd deploy the Function using automated CLI commands. These scripts are optional. You can perform all the steps in these scripts manually, using the Azure CLI or the Azure Portal.
```
Comment (sebastianburckhardt) [inline docs/hello-sample.md:73]: nit: amd -> and

**Review by sebastianburckhardt — CHANGES_REQUESTED** [review]
Looks great, a big improvement. Love the unicode characters you added: 🚀!

I have some very minor suggestions and found one typo, other than that I think this is ready.

File: docs/hello-sample.md (hunk 1 above)
Comment (lechnerc77) [reply docs/hello-sample.md:3]: Fair point.

File: docs/hello-sample.md (hunk 5 above)
Comment (lechnerc77) [reply docs/hello-sample.md:73]: Corrected ... there have been several "amd" in there thanks to copy&paste ;-)

File: docs/hello-sample.md (hunk 2 above)
Comment (lechnerc77) [reply docs/hello-sample.md:29]: Added that as a remark

**Comment by lechnerc77** [comment]
Made corrections as suggested.

**Review by sebastianburckhardt — APPROVED** [review]

**Pull request merged**